app/controllers/api/v1x0/requests_controller.rb

```diff
@@ -4,6 +4,11 @@ class RequestsController < ApplicationController
       include Mixins::IndexMixin
       include Mixins::RBACMixin
 
+      PERSONA_HEADER    = 'x-rh-persona'.freeze
```

Comment:
@bzwei this should be header key
PERSONA_HEADER_KEY
And should that go in here
https://github.com/ManageIQ/manageiq-api-common/blob/83aa89d7049de6e7864750bee980c04cdc5f9667/lib/manageiq/api/common/request.rb#L12

Reply:
https://github.com/ManageIQ/manageiq-api-common/pull/109 has been created for this purpose. Shall we wait until it is merged or make a follow-up PR to reference it?